source/sitemap.xml

```diff
@@ -0,0 +1,55 @@
+---
+use: ["posts", "sysadmin", "developer", "designer"]
+permalink: sitemap.xml
+---
+<?xml version="1.0" encoding="UTF-8"?>
+<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
+    <url>
+        <loc>{{ site.url }}</loc>
+        <lastmod>{{ site.calculated_date | date('Y-m-d') }}</lastmod>
+        <changefreq>daily</changefreq>
+        <priority>0.8</priority>
+    </url>
+    {% for post in data.posts %}
+    <url>
+        <loc>{{ site.url }}{{ post.url }}</loc>
+        <lastmod>{{ post.date|date('c') }}</lastmod>
+        <changefreq>weekly</changefreq>
+        <priority>1.0</priority>
+    </url>
+    {% endfor %}
+
+    {% for page in data.sysadmin %}
+        {% if page.github_link %}
```

Comment:
Why do you test for `github_link` here? Wouldn't  `indexed` not be more appropriate?